### PR TITLE
Load testing/ra

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/test/load-tests?view=aspnetcore-1.0&branch=pr-en-us-10262)
PR of #10208
@jeremymeng Can you review my [commit](https://github.com/aspnet/Docs/pull/10262/commits/0864494ff7fca3bacdbdcde4e6461f652bcf46c4)?

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@jeremymeng 	Not sure of my rewrite of "The subject under testing is running under normal conditions."

**Comment by jeremymeng** [comment]
@Rick-Anderson Looks much better! Thanks!

File: aspnetcore/test/load-tests.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+Load testing and stress testing are important to ensure a web app is performant and scalable. Their goals are different even they often share similar tests.
+
+**Load tests**: Tests whether the app can handle a specified load of users for a certain scenario while still satisfying the response goal. The app is run under normal conditions.
+
+**Stress tests**: Tests app stability when running under extreme conditions and often a long period of time:
+
+* High user load – either spikes or gradually increasing.
+* Limited computing resources.  
+
+Under stress, can the app recover from failure and gracefully return to expected behavior? The app is run under normal conditions.
```
Comment (mjrousos) [inline aspnetcore/test/load-tests.md]: Is this supposed to say "The app is run under normal conditions" or is that a copy/paste error from the **Load tests** part. It sounds like stress test conditions are *not* normal.

**Comment by mjrousos** [comment]
Looks good!

File: aspnetcore/test/load-tests.md (hunk 1 above)
Comment (jeremymeng) [reply aspnetcore/test/load-tests.md]: @Rick-Anderson yes, under stress test, the app is running under unusual conditions.

**Pull request merged**